src/redux/missions/missions.js

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios';
+
+const FETCH_MISSIONS = 'bookstore/src/redux/books/FETCH_MISSIONS';
```

Comment:
I think naming the folder and files with names relating to the project will be preferred for clarity.
Eg 'spaceTravellersHub/src/redux/missions/FETCH_MISSIONS'